time[15]    stamp
float64[15] left_wheel_angle
float64[15] right_wheel_angle
float64[15] steering_angle
uint16 nb_data